app/services/idv/session.rb

```diff
@@ -37,13 +40,13 @@ def respond_to_missing?(method_sym, include_private)
     end
 
     def proofing_started?
-      resolution.present? && applicant.present? && resolution.success?
+      resolution.present? && applicant.present? && resolution_successful
```

Comment:
Ideally, yes, but the way this class is structured, this is an attr_writer instance attribute, not just a private method. This value first gets set in `ProfileStep` (`idv_session.resolution_successful = proofer_resolution.success?`), and is then read in the Session class.

I have plans to refactor this Session class later.